pkgs/servers/computing/slurm/default.nix

```diff
@@ -35,15 +35,16 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkgconfig libtool ];
   buildInputs = [
     curl python munge perl pam openssl
-      mysql.connector-c ncurses gtk2
+      mysql.connector-c ncurses gtk2 lz4
       lua hwloc numactl readline freeipmi
   ] ++ stdenv.lib.optionals enableX11 [ libssh2 xorg.xauth ];
 
   configureFlags = with stdenv.lib;
-    [ "--with-munge=${munge}"
-      "--with-ssl=${openssl.dev}"
+    [ "--with-freeipmi=${freeipmi}"
       "--with-hwloc=${hwloc.dev}"
-      "--with-freeipmi=${freeipmi}"
+      "--with-lz4=${lz4}"
```

Comment:
Using the dev output here works: `--with-lz4=${lz4.dev}`. 
Would you mind also adding `"--with-zlib=${zlib}"` ?